apps/client/src/login/types.tsx

```diff
@@ -0,0 +1,3 @@
+export interface LoginFormProps {
+  handleSubmit?: (event: React.FormEvent<HTMLFormElement>) => void;
```

Comment:
Not blocking: Why is `handleSubmit` optional? Is there a situation when `<LoginForm />` is used without `handleSubmit` defined?